lage.config.js

```diff
@@ -1,15 +1,14 @@
 module.exports = {
   npmClient: 'yarn',
   pipeline: {
-    ['build-tools']: ['^build-tools'],
-    build: ['build-tools', '^build'],
+    build: ['^build'],
     buildci: ['build', 'test', 'depcheck'],
-    bundle: ['build-tools', 'build'],
+    bundle: ['build'],
     clean: [],
-    depcheck: ['build-tools', 'align-deps'],
-    lint: ['build-tools'],
-    prettier: ['build-tools'],
-    ['prettier-fix']: ['build-tools'],
-    test: ['build-tools', 'lint', 'build'],
+    depcheck: ['align-deps'],
```

Comment:
Maybe a follow up, but these are technically 2 different verbs. The combination is a carryover from when align deps was named depchek except we had a linter named dep check already. Could we separate the verbs?